portfolio/src/main/webapp/styles/main-page.css

```diff
@@ -64,6 +23,7 @@ body {
     transition: all 1000ms;
 }
 
+
```

Comment:
remove one blank line

Reply:
Now only to pick up this extra line & give it to the other files that need it haha.